src/containers/taskList.jsx

```diff
@@ -128,16 +151,28 @@ const TaskList = ({ style = {}, autoSelect = false }) => {
       }
     }
 
+    const resolveActiveStyle = () => isActive ? {opacity: 1} : {opacity: 0.5}
```

Comment:
The only reason why I did that was because it is being used twice - for `iconStyle` and for `textStyle`. But yeah, it can be used directly as ternary since it is simple opacity switch.

Reply:
Sure, makes sense, done.